deployed-devices/rest/certificates/create-certificate/output/create-certificate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sid": "CYXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "friendly_name": null,
+  "fleet_sid": "FLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "account_sid": "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "device_sid": "THXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "thumbprint": "00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff",
```

Comment:
Is this thumbprint a static string? Something users know or generate? If the string here is a placeholder for something variable, it might be good to use a string like 'your_generated_thumbprint' instead. If not, carry on! 

(there are a few other places this occurs - if you do change this string, just double-check that it's the same across the board. thanks!)